packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -308,12 +308,24 @@ export class NodeHttpRequestResponseHandler implements NodeHttpStreamsHandler {
 
         nodeHttpStreams.responseStream.end();
 
+        let bodyToLog = '';
+
+        // Set body to string '<Buffer>' if it is a Buffer Object to not pollute logs
+        if (response.body && response.body instanceof Buffer) {
+
+          bodyToLog = '<Buffer>';
```

Comment:
can't we convert the `Buffer` to string + `.slice(0, 2000)` ?

Reply:
We could but I am not sure we want that.

- Just from useid's perspective `<Buffer>` would actually be better, atm only id-proxy ever returns a Buffer for a body because we are getting that from Auth0/our upstream, and it's convenient that that doesn't end up in the logs. (we should verify this, @AbelVandenBriel ping)
- also, we would be parseing to Buffer and from Buffer and I honestly don't know what large objects would do performance/memory wise.